.github/workflows/lint.yml

```diff
@@ -20,8 +20,8 @@ jobs:
         with:
           go-version: "1.22.x"
 
-      - name: Run lint-prepare
-        run: make lint-prepare
+      - name: Run tool
```

Comment:
Should it be `Run tools`? It looks a bit inconsistent. Or maybe `Install tools`?